version.php

```diff
@@ -24,7 +24,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2015051800;        // The current plugin version (Date: YYYYMMDDXX)
+$plugin->version   = 2021030900;        // The current plugin version (Date: YYYYMMDDXX)
```

Comment:
Can you please also get github actions working, this can be a new pr if you prefer

Reply:
I'll set that up in a different pull.